config/README.md

```diff
@@ -69,24 +69,23 @@ As an example, this is the 2nd node (offset 1) in a set of 4:
         -o /opt/libra/etc \
         -s 0123456789abcdef101112131415161718191a1b1c1d1e1f2021222324252627
 
-To create a mint service's consensus peer config and key that connects to
-this service:
+To create a mint service's key:
 
-    faucet-config-builder \
-        -n 4 \
+    config-builder faucet \
```

Comment:
We are still using `faucet-config-builder` in Faucet's dockerfile (https://github.com/libra/libra/blob/588bfcce69eba88351c7000b1d668184efbfa9b6/docker/mint/mint.Dockerfile#L37).  Do we need a corresponding there?

Reply:
Good call. I wanted to work with the owners of the respective tools to help evaluate their use cases. The old faucet builder dumped a consensus peers file that is no longer relevant, for example, but may still be expected to exist for scripts to execute correctly.

This PR only deprecated the tools but has yet to delete them until I get the okay from @opsguy and @mgorven